2.6.0/spec/accesses.md

```diff
@@ -395,6 +397,14 @@ Specifies whether the end customer is required to have a separate contract with
  * Data format: One of REQUIRED_NOT_VALID, REQUIRED_VALID, NOT_REQUIRED or empty string ""
  * Mandatory 
 
+### nationalAddressUUID
+
+Universally Unique Identifier (UUID) for the Address. May differ from country to country depending on national preferences. 
+Sweden: PTS (The Swedish Post and Telecom Authority) uses the [Lantmäteriet Real Property UUID](https://www.lantmateriet.se/en/geodata/geodata-products/product-list/real-property-information-download/)
+
+ * Data format: [uuid](../common/dataformats.md#uuid)
+ * Required
```

Comment:
If this specification is to be used outside of Sweden (as alluded to in the text above) there is no guarantee that there is a national address id and especially not that it is in the format of a UUID. Should this field only be required for Swedish addresses?

```md
* Required for addresses in Sweden
```

Reply:
I was uncertain of the dataformat as well. Maybe make the dataformat a string? Then note what more specific dataformat the string is for different countries?

* Data format: string (max 256 chars) and depends on country.
* Sweden: Data format UUID

Reply:
We have at least a few other data fields (e.g. appartment number) with Sweden-specific formats.
I suggest that internationalization is a focus for a future version. (Netadmin has extensive thoughts and fields that are necessary in other countries.)